sdstore/md5helper.go

```diff
@@ -183,48 +196,27 @@ param - path		file or folder path
 return - md5map / error	success - return md5map; error - return error description
 */
 func GenerateMd5(path string) (map[string]string, error) {
-	var wg sync.WaitGroup
-
-	const MaxConcurrencyLimit = 3000
+	var rwm sync.RWMutex
 	md5Map := make(map[string]string)
-	md5Channel := make(chan md5Hash)
-	defer close(md5Channel)
+	maxGoRoutines, _ := strconv.Atoi(getEnv("SD_CACHE_MAX_GO_WORKERS", "10000"))
```

Comment:
so 10000 is the no. of workers we are setting?

Reply:
yes, updated the env variable name to SD_CACHE_MAX_GO_THREADS